workflow/snakemake_rules/main_workflow.smk

```diff
@@ -20,74 +20,45 @@ rule combine_input_metadata:
         python3 scripts/combine_metadata.py --metadata {input.metadata} --origins {params.origins} --output {output.metadata} 2>&1 | tee {log}
         """
 
-if "use_nextalign" in config and config["use_nextalign"]:
-    rule align:
-        message:
-            """
-            Aligning sequences to {input.reference}
-              - gaps relative to reference are considered real
-            """
-        input:
-            sequences = lambda wildcards: _get_path_for_input("sequences", wildcards.origin),
-            genemap = config["files"]["annotation"],
-            reference = config["files"]["alignment_reference"]
-        output:
-            alignment = "results/aligned{origin}.fasta",
-            insertions = "results/insertions{origin}.tsv",
-            translations = expand("results/translations/seqs{{origin}}.gene.{gene}.fasta", gene=config.get('genes', ['S']))
-        params:
-            outdir = "results/translations",
-            genes = ','.join(config.get('genes', ['S'])),
-            basename = "seqs{origin}"
-        log:
-            "logs/align{origin}.txt"
-        benchmark:
-            "benchmarks/align{origin}.txt"
-        conda: config["conda_environment"]
-        threads: 8
-        resources:
-            mem_mb=3000
-        shell:
-            """
-            nextalign \
-                --jobs={threads} \
-                --reference {input.reference} \
-                --genemap {input.genemap} \
-                --genes {params.genes} \
-                --sequences {input.sequences} \
-                --output-dir {params.outdir} \
-                --output-basename {params.basename} \
-                --output-fasta {output.alignment} \
-                --output-insertions {output.insertions} > {log} 2>&1
-            """
-else:
-    rule align:
-        message:
-            """
-            Aligning sequences from {input.sequences} to {input.reference}
+rule align:
+    message:
+        """
+        Aligning sequences to {input.reference}
```

Comment:
Good choice to move to `nextalign` for everything. If `nextalign` is missing from the environment we get the error
```
/usr/local/bin/bash: line 1: nextalign: command not found
```
which is probably good enough. I considered adding something simple to the `Snakefile` like the following, but it may be more trouble than it's worth.
```
# Ensure `nextalign` is available, as we no longer support `mafft`
import shutil
if shutil.which("nextalign") is None:
    logger.error("ERROR: `nextalign` does not seem to be installed, which is needed for this pipeline.")
    sys.exit(1)
```

Reply:
Yeah, I think we can't handle this much better than recommending users add the `--use-conda` to their Snakemake commands or use the CLI and Docker image.